sass/components/rtl_support/_badges.scss

```diff
@@ -0,0 +1,17 @@
+// RTL Badges
+.rtl {
+	span.badge {
+		margin-left: initial;
+		margin-right: 14px;
+		float: left;
+
+		&.new:after {
+			content: " جدید";
```

Comment:
@tomscholz Persian, Arabic and Hebrew are RTL for sure.
@NonameSLdev I removed that line. Maybe That "new" content for badge shouldn't be included at all. It seems more flexible if it was fetched as a data-attrib in CSS like this: content: attr(data-badge-text);